packages/didier-stevens-beta.vm/tools/chocolateyinstall.ps1

```diff
@@ -17,15 +17,16 @@ try {
     $toolDir = Get-Item "${Env:RAW_TOOLS_DIR}\Beta-*"
     VM-Assert-Path $toolDir
 
-    # Add shortcut for commonly used office tools
+    # Add shortcut for commonly used office python tools
     ForEach ($toolName in @('onedump')) {
-      $executablePath = Join-Path $toolDir "$toolName.py"
-      VM-Install-Shortcut $toolName $category $executablePath -consoleApp $true -arguments "--help"
+      $executablePath = (Get-Command python).Source
```

Comment:
The shortcut is not broken at the moment, executing the `.py` file directory works well:
![Screenshot 2024-02-19 at 08 17 19](https://github.com/mandiant/VM-Packages/assets/16052290/9d8c91ca-9467-4ff3-9629-15fa32e5f277)

I would prefer to change only the icon (to be the python one) instead of changing the way we execute the tool.